css/style.css

```diff
@@ -464,10 +496,270 @@ div .taggs {
+  }
+
+  /* ////// Projects /////// */
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem 0;
+  }
+
+  .work-page-left {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    align-content: center;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+  }
+
+  .work-page-right {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    background: #fff;
+    border: none;
+    border-radius: 24px;
+    padding: 2rem;
+  }
+
+  .card1-content,
+  .card3-content {
+    margin: 30px 20px 0 0;
+  }
+
+  .card2-content {
+    grid-row: 1/-2;
+    margin: 30px 20px 0 0;
+    padding-left: 15px;
+  }
+
+  .work-title {
+    width: 100%;
+  }
+
+  .card1-image,
+  .card2-image,
+  .card3-image {
+    transition: transform 0.3s;
+  }
+
+  .card1-image:hover,
+  .card2-image:hover,
+  .card3-image:hover {
+    transform: scale(1.07);
+  }
+
+  .clients {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    padding: 0 15px;
+    margin: 1px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  div .taggs {
+    float: right;
+    width: 40px;
+    height: 20px;
+    padding-bottom: 20px;
+  }
+
+  .tag1 {
+    width: 50px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    margin-bottom: 20px;
+    color: #396df2;
+  }
+
+  .tag2 {
+    width: 40px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    color: #396df2;
+  }
+
+  .tag3 {
+    width: 80px;
+    height: 20px;
+    margin-right: 8px;
+    border: none;
+    border-radius: 6px;
+    color: #396df2;
+  }
+
+  /* //////// About-myself-desktop//////// */
+
+  .about-myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto;
+    background: #fff;
+    border-top-right-radius: 6rem;
+  }
+
+  .about-myself-content {
+    padding: 5rem 2rem 5rem 6.5rem;
+  }
+
+  .about-myself-languages {
+    padding: 2.5rem 3rem 5rem 0;
+    margin-top: 4rem;
+  }
+
+  .languages {
+    padding-left: 0;
+    padding-bottom: 0.6rem;
+    color: #344563;
+  }
+
+  .framework-skills-desktop {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .frameworks {
+    display: flex;
+    padding: 0.5rem 4rem 0.5rem 0;
+  }
+
+  .framework1 {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  hr {
+    margin-left: 1rem;
+  }
+
+  .html-skill,
+  .css-skill {
+    margin-right: 0.8rem;
+  }
+
+  .framework-skills {
+    display: none;
+  }
+
+  .greater-tag {
+    margin-right: 0;
+  }
+
+  /* //////// Contact ///////// */
+
+  .contact {
+    width: 100%;
+    grid-column: 1/-1;
+    display: flex;
+    justify-content: center;
+    background-image: url(/images/desktop_main/contact_desktop.svg);
```

Comment:
- [ ]  This background image is not showing up in the browser. kindly consider using .png file. this background image should be there because we must stick to the Figma design.
your design
![ty](https://user-images.githubusercontent.com/96683843/213529982-fbaf9e52-c8c8-4baa-a6b1-aea44e4c7031.jpg)
figma design
![asd](https://user-images.githubusercontent.com/96683843/213530038-b48f950a-5193-41b2-aea0-e694586865ec.jpg)